### DISCUSSION TITLE
Synchronous import of components exported by module federation？

### DISCUSSION THREAD

**Discussion by homestylew** [webpack/webpack#18794] [Q&A]
```javascript
// 1.component.tsx
export { default as TestA } from "../components/TestA";
export { default as TestB } from "../components/TestB";

// 2.exposes all components
exposes: {
          './DyCom': path.resolve(__dirname, './src/component.tsx'),
      },
// 3.exposes all components
import { TestA ,TestB} from "xxx/DyCom";
```
Synchronous import of components exported by module federation？
webpack doc：A chunk loading operation is usually an import() call, but older constructs like require.ensure or require([...]) are supported as well.

can I like this?
import { TestA ,TestB} from "xxx/DyCom" ;

**Comment by alexander-akait** [comment]
> but older constructs like require.ensure or require([...]) are supported as well.

There are no `require.ensure` and `require` in browsers
